introduction_to_amazon_algorithms/jumpstart_image_classification/Amazon_JumpStart_Image_Classification.ipynb

```diff
@@ -2,15 +2,15 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        max_run=360000,
</pre><p>How long is the runtime on this training with this high max run?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3429/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>It takes about 10 minutes. Max_run is kept large just to cover all the edge cases.</p>